mailpoet/assets/js/src/newsletters/listings/standard.jsx

```diff
@@ -187,6 +187,25 @@ class NewsletterListStandardComponent extends Component {
     };
   }
 
+  componentDidMount() {
```

Comment:
I opted not to refactor this file to TSX as I already spent way more time than I wanted on this ticket. I tried to do the refactor but ran into some issues that were not straightforward to fix and decided to leave it to another moment.